modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/ObjectDefinitionRESTContextPathRegistry.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.object.rest.internal;
+
+import com.liferay.object.model.ObjectDefinition;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Javier de Arcos
+ */
+@Component(
+	immediate = true, service = ObjectDefinitionRESTContextPathRegistry.class
```

Comment:
Does this need an immediate?

Reply:
🤔  I guess not, as it would be created when it is injected from the Deployer. I will change it